ui/app/components/clients/config.js

```diff
@@ -43,38 +44,35 @@ export default class ConfigComponent extends Component {
   }
 
   get modalTitle() {
-    let content = 'Turn usage tracking off?';
-    if (this.args.model && this.args.model.enabled === 'On') {
-      content = 'Turn usage tracking on?';
-    }
-    return content;
+    return `Turn usage tracking ${this.args.model.enabled.toLowerCase()}?`;
   }
 
   @(task(function* () {
     try {
       yield this.args.model.save();
+      this.router.transitionTo('vault.cluster.clients.config');
     } catch (err) {
       this.error = err.message;
-      return;
     }
-    this.router.transitionTo('vault.cluster.clients.config');
```

Comment:
It looks like moving this to inside the `try` might fix the issue of the API error appearing while the modal is still open? Something I remembered seeing when Mike was sharing his screen during sync

Reply:
Oh I don't think I was aware of that but I moved it up in any case just because the code flow in it's original state seemed a bit strange to me!

Reply:
Yeah, I agree. It'd probably be worth testing that the modal closes when an API error is thrown?

Reply:
Oh yes good call it would remain open. I found while testing that `error` was not a tracked property so the message wasn't showing either. I'll get those fixed up. Thanks for pointing this out!